timescaledb/overview/page-index/page-index.js

```diff
@@ -83,6 +83,13 @@ module.exports = [
             keywords: ['install', 'TimescaleDB', 'Timescale Forge', 'Timescale Cloud', 'self-hosted'],
             excerpt: "Deployment options for TimescaleDB",
           },
+          {
+            title: "TimescaleDB on Kubernetes",
+            href: "timescale-kubernetes",
+            tags: ['install', 'timescaledb', 'self-hosted', 'Kubernetes', 'Helm Charts'],
+            keywords: ['install', 'TimescaleDB', 'self-hosted', 'Kubernetes'],
+            excerpt: "Deployment options for TimescaleDB",
+          },
```

Comment:
Is this the right location for this page? It feels out of place (not having other deployment options with their own pages.

IMO, either make it a child page of "deployment options", or add it to the main "deployment options" page for now with links to the install doc.

Reply:
@Loquacity could you please suggest a place for this page because i see two deployment-options.md in the docs.
<img width="371" alt="Screenshot 2022-03-10 at 3 09 57 PM" src="https://user-images.githubusercontent.com/37059749/157634560-a8952efd-fe25-4716-9cd9-2cb20a9035f3.png">

Reply:
Both of those pages are identical, but only one of them is published (this one: timescaledb/overview/deployment-options.md). That means we should delete timescaledb/overview/core-concepts/deployment-options.md. That said, I'm not sure why we should bury this info in the Overview. @ryanbooz can you please elaborate on your reasoning here?